guides/common/modules/con_using-ipxe-to-reduce-provisioning-times.adoc

```diff
@@ -0,0 +1,12 @@
+[id="using-ipxe-to-reduce-provisioning-times_{context}"]
+= Using iPXE to reduce provisioning times
```

Comment:
It would be great to make the titles more specific to make it clear you're talking about Project.

* Using iPXE to reduce provisioning times _with Project_
* Prerequisites for using iPXE _with Project_
* Configuring iPXE environment for use _with Project_

On one hand, this is redundant because the whole book is about Satellite. But we don't want to think about docs in terms of books; we want to transition from the book-based narrative to topic-based narrative, and in that framework, you need more specific titles (because Every Page Is Page One).

Reply:
I'm not sure about this, it looks a bit too much to repeat it for every module.